src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Use `name()` for getting String representation of enum constants.
FYI, don’t use `toString()` or `String.valueOf()`(it will call toString() under the hood) for getting String representation of enum constants. Because toString() is common for all enum constants.
If you override this method like below:

```
 @Override
  public String() toString() {
     return "default";
}
```

 then for every constant `toString()` will be returning default, that’s not ok. So it’s better to use standard method of enum `name()` that will be returning always String representation of concrete enum constant.